README.md

```diff
@@ -30,7 +30,22 @@ Pravega Presto connector has a set of unit tests that can take several minutes t
 
     ./gradlew test
 
-## Running Presto
+## Installing Presto
+
+Instructions for downloading, installing and configuring Presto can be found here: https://prestodb.io/docs/current/installation/deployment.html.
+
+A typical Presto installation installs the Presto server in /usr/lib. You download (via wget) the gzip'd tar file from Maven using the link defined in the PrestoDB deployment section.  Untar the file while in /usr/lib, which will create a presto-server-<RELEASE> directory.  Create a symbolic link to the code as follows
```

Comment:
When following the instructions in:

https://prestodb.io/docs/current/installation/deployment.html

I actually downloaded a tgz, which I unpacked into a directory of my choosing, say `fpj/presto` to illustrate. For the following comments, let me call it `$PRESTO_HOME`.

Reply:
By typical I meant, that's where the presto rpm installs the package, in /usr/lib/presto, and the config location is /etc/presto.  When using a tar file, you can untar it anywhere, yes.  I'll make it more flexible.